include/ignition/math/config.hh

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#ifndef IGNITION_MATH__CONFIG_HH_
+#define IGNITION_MATH__CONFIG_HH_
+
+#include <gz/math/config.hh>
+
+/* Version number */
+// #define IGNITION_MATH_MAJOR_VERSION GZ_MATH_MAJOR_VERSION
+// #define IGNITION_MATH_MINOR_VERSION GZ_MATH_MINOR_VERSION
+// #define IGNITION_MATH_PATCH_VERSION GZ_MATH_PATCH_VERSION
+
+// #define IGNITION_MATH_VERSION GZ_MATH_VERSION
+// #define IGNITION_MATH_VERSION_FULL GZ_MATH_VERSION_FULL
+
+// #define IGNITION_MATH_VERSION_NAMESPACE GZ_MATH_VERSION_NAMESPACE
+
+// #define IGNITION_MATH_VERSION_HEADER GZ_MATH_VERSION_HEADER
+
+/* #undef IGNITION_MATH_BUILD_TYPE_PROFILE */
+/* #undef IGNITION_MATH_BUILD_TYPE_DEBUG */
+/* #undef IGNITION_MATH_BUILD_TYPE_RELEASE */
+
+namespace ignition
+{
+}
+
+namespace gz
+{
+  using namespace ignition;
+}
```

Comment:
Do we need to repeat this here? It's already including `gz/math/config.hh`

Reply:
Ah, you're right. It's a leftover from when I wasn't including it on the `gz` one... :sweat: 
I need to remove them from the entire stack